project/src/store/actions/city-actions/city-actions.ts

```diff
@@ -0,0 +1,7 @@
+import { createAction } from '@reduxjs/toolkit';
+
+import OfferType from '../../../types/offerType';
+
+export const changeCityName = createAction<string>('city/changeCityName');
```

Comment:
Желательно использовать enum или константу const ACTIONS = { actionName: "" }  для экшенов
```suggestion
export const changeCityName = createAction<string>(Actions.CHANGE_CITY_NAME);
```